python/cdp/evm_server_account.py

```diff
@@ -737,7 +737,7 @@ async def __experimental_use_network__(
         """
         from cdp.network_scoped_evm_server_account import NetworkScopedEvmServerAccount
 
-        return NetworkScopedEvmServerAccount(self, network, rpc_url)
+        return NetworkScopedEvmServerAccount(self, network)
```

Comment:
You also need to remove `rpc_url` from the arguments in `__experimental_use_network__`